package.json

```diff
@@ -15,7 +15,7 @@
     "start": "yarn react-native start",
     "test": "node node_modules/jest/bin/jest.js --config=.jest.json",
     "test-ci": "./scripts/test.sh",
-    "postinstall": "node ./scripts/configureApplication.js"
+    "postinstall": "node ./scripts/configureApplication.js && sed -i '' 's\/#import <fishhook\\/fishhook.h>\/#import \"fishhook.h\"\/' ./node_modules/react-native/Libraries/WebSocket/RCTReconnectingWebSocket.m && sed -i '' 's\/#import <RCTAnimation\\/RCTValueAnimatedNode.h>\/#import \"RCTValueAnimatedNode.h\"\/' ./node_modules/react-native/Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.h"
```

Comment:
could this rather be added to the configureApplication script?
`const { exec } = require('child_process');`
`exec('sed -i ...')`

Reply:
I can tweak these minor issues during integration. The same applies to squashing all commits and similar. We just need to have a working example and not copy-paste... 😞

Reply:
Thanks, I wasn't aware of `configureApplication` script... 

Is there anything else we need to do or another method to add? In other case I will create `pod`.., but then adding methods will be more complicated. @mrtnzlml @tbergq

Reply:
Probably no - please extract it into our private `react-native-native-modules` repo and require it here.

Reply:
Debugging menu and code reloading needs to work

Reply:
@tbergq I don't know if I can help you with that. Do you have any thoughts what is wrong?

Reply:
I can try to look into it tomorrow. I don't know why. I cloned your repo, tested from your master branch, and there it works, in the ios-logging-module-dev it does not. So there must be something in this PR that does it.

Reply:
The problem is that I don't know what debug menu is and how to open it. 
Should I read https://facebook.github.io/react-native/docs/debugging.html or is there any better docs?

Reply:
If you are on a fysical device, shake it, if you are on simulator, click on hardware => shake gesture. It should look like this: 

<img width="414" alt="skjermbilde 2018-02-07 kl 10 14 25" src="https://user-images.githubusercontent.com/7705041/35907989-bf64ecd2-0bef-11e8-8160-a6ae186507ed.png">